index.js

```diff
@@ -4,12 +4,22 @@ let form = document.querySelector("form");
 form.addEventListener("submit", function (event) {
   event.preventDefault();
 
-  console.log("Form submitted");
+
+  // Store the user's email address as userEmail (string/text)
+  let userEmail = document.querySelector("#email").value;
+
+  // Get the user's level - UserLevel (string)
+  let userLevel = document.querySelector("#level").value;
+  
+  // Get the user's hours of study UserHours (number)
+  let userHours = parseInt(document.querySelector("#hoursPerWeek").value);
+
+   // Validate the user's input
+
+    console.log({ errors });
```

Comment:
This `console.log()` will throw an error, as you have moved the `errors` object into your `validateForm()` function.